lib/spack/external/__init__.py

```diff
@@ -12,20 +12,23 @@
 * Homepage: https://altgraph.readthedocs.io/en/latest/index.html
 * Usage: dependency of macholib
 * Version: 0.17.3
+* License: MIT
 
 archspec
 --------
 
 * Homepage: https://pypi.python.org/pypi/archspec
 * Usage: Labeling, comparison and detection of microarchitectures
 * Version: 0.2.5-dev (commit 7e6740012b897ae4a950f0bba7e9726b767e921f)
+* License: Apache-2.0
```

Comment:
This one is Apache-2.0 or MIT (same as Spack)

Reply:
Update both here and in `py-archspec`.